doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,187 @@
+# Mount a Repo to a Local Computer
+
+Sometimes you might need to access files directly in
+a Pachyderm repository without downloading them first
+to your local computer. Pachyderm enables
```

Comment:
This is not the use case for mount. Mount does not prevent you from needing to download the data to your local computer. It simply allows you to lazily download data -- meaning it only downloads the files when you try to access them. If you are accessing all the files, it's actually slower than just get file although it's convenient and users are often fairly familar with mounting file systems